### PR TITLE
New keybindings - Issue #73

### PR DESCRIPTION
**Pull request by sinnaj-r**

### CODE REVIEW

File: packages/Algernon.package/ALGCore.class/class/initializeGlobalKeyBindings.st [hunk 1]
```diff
@@ -4,7 +4,7 @@ initializeGlobalKeyBindings
 	KeyBindings := Dictionary newFrom:
 		{ '<down>' -> [:anEvent :morph | morph selectNext]. 
 		'<up>' -> [:anEvent :morph | morph selectPrevious]. 
-		'<right>' -> [:anEvent :morph |  morph showChildList]. 
-		'<left>' -> [:anEvent :morph | morph showParentList]. 
+		'<tab>' -> [:anEvent :morph |  morph showChildList]. 
+		'<Shift-tab>' -> [:anEvent :morph | morph showParentList].
```
Comment (BenBals) [inline packages/Algernon.package/ALGCore.class/class/initializeGlobalKeyBindings.st:8]: Must Shift be capitalized?

**Review by BenBals — APPROVED** [review]
Looks good. I mean it's not that much code.

File: packages/Algernon.package/ALGCore.class/class/initializeGlobalKeyBindings.st (hunk 1 above)
Comment (sinnaj-r) [reply packages/Algernon.package/ALGCore.class/class/initializeGlobalKeyBindings.st:8]: At least its capitalized in the original event.

Comment (BenBals) [reply packages/Algernon.package/ALGCore.class/class/initializeGlobalKeyBindings.st:8]: Okay, thanks.

**Review by robertkirsten — APPROVED** [review]
Looks good to me too.

**Pull request merged**